benchmarks/coremark/add_init_message.patch

```diff
@@ -0,0 +1,12 @@
+--- coremark/posix/core_portme.c
++++ coremark/posix/core_portme.c
+@@ -208,6 +208,9 @@ ee_u32 default_num_contexts = MULTITHREAD;
+ void
+ portable_init(core_portable *p, int *argc, char *argv[])
+ {
++
++    ee_printf("Running CoreMark...\n");
```

Comment:
why need add this?

Reply:
To be able to tell that the application has started when running as the entry point.
By default it doesn't show anything and, when testing, it looked like it was stuck when using many iterations.

Reply:
so, please upstream to coremark communtiy.